Phaethon/WebClient/Controllers/InvoiceController.cs

```diff
@@ -28,6 +28,7 @@ public ActionResult Index()
         }
 
         [HttpGet]
+        [Route("Invoice/Edit/{id:int}")]
```

Comment:
Are you sure you want this?
I'd put it string (to make sure it always reaches my endpoint) and then just check it and return StatusCode.BadRequest with user friendly error message, or just convert it.

Reply:
this has to be done as there are 2 get methods with same name, othervise it doesnt work (atleast as much as i tried i couldnt make it work othervise, so i made this solution)